app/src/common/shared/org/mozilla/vrbrowser/browser/SessionStore.java

```diff
@@ -1042,7 +1042,9 @@ public void onCanGoForward(@NonNull GeckoSession aSession, boolean aCanGoForward
     private String checkYoutubeOverride(String aUri) {
         try {
             Uri uri = Uri.parse(aUri);
-            if (!uri.getHost().toLowerCase().contains("www.youtube.")) {
+            String hostLower = !uri.getHost().toLowerCase();
```

Comment:
Typo here ~!~

Reply:
thanks (been running this WebExt on desktop for rapider development)